aurora/model/aurora.py

```diff
@@ -84,7 +84,7 @@ def __init__(
         dynamic_vars: bool = False,
         atmos_static_vars: bool = False,
         separate_perceiver: tuple[str, ...] = (),
-        modulation_head: bool = False,
+        modulation_heads: Optional[tuple[str, ...]] = None,
```

Comment:
```suggestion
        modulation_heads: tuple[str, ...] = (),
```

Shall we set the default value to an empty tuple to be consistent with the other keyword args? That slightly simplifies the implementation as `modulation_heads` will always be iterable.